pom.xml

```diff
@@ -1224,12 +1224,12 @@ under the License.
 					<plugin>
 						<groupId>org.apache.maven.plugins</groupId>
 						<artifactId>maven-source-plugin</artifactId>
-						<version>2.2.1</version><!--$NO-MVN-MAN-VER$-->
```

Comment:
is there any particular reason for the upgrade?

Reply:
No; it just felt like a good opportunity to do so since we're looking at the plugin anyway.